db_functions.py

```diff
@@ -234,8 +239,8 @@ def nabidky_spolujizdy(id_zavodu):
             conn.close()
 
 
-def vyber_spolujizdu(id_jizdy):
-    """ Vybere konkretni id jizdy pro spolujizdu """
+def choose_carpool(id_race):
```

Comment:
This should be `id_ride` instead of `id_race` as well? Also fix it below in this function.